src/Response/AuthorizationResponse.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\BoxNowBundle\Response;
+
+use Webmozart\Assert\Assert;
+
+class AuthorizationResponse implements ResponseInterface
+{
+    private string $accessToken;
+
+    private string $tokenType;
+
+    private int $expiresIn;
+
+    public function __construct(private readonly array $data)
+    {
+        $this->handleData();
```

Comment:
are you need property $data here? Passing this to `handleData` method seems enough